.github/workflows/release.yml

```diff
@@ -360,8 +360,8 @@ jobs:
         if: github.event_name == 'push'
         with:
           allowUpdates: true
-          prerelease: ${{ env.STABLE_RELEASE == 'false' }}
```

Comment:
should we also drop https://github.com/TabbyML/tabby/pull/3613/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34L348-L358 since STABLE_RELEASE is no longer used

Reply:
Done